specification/sql/resource-manager/Microsoft.Sql/preview/2021-08-01-preview/SyncGroups.json

```diff
@@ -0,0 +1,1065 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get a hub database schema.": {
+            "$ref": "./examples/SyncGroupGetHubSchema.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/syncGroups/{syncGroupName}/logs": {
+      "get": {
+        "tags": [
+          "SyncGroups"
+        ],
+        "description": "Gets a collection of sync group logs.",
+        "operationId": "SyncGroups_ListLogs",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "description": "The name of the database on which the sync group is hosted.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "syncGroupName",
+            "in": "path",
+            "description": "The name of the sync group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "startTime",
+            "in": "query",
+            "description": "Get logs generated after this time.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "endTime",
+            "in": "query",
+            "description": "Get logs generated before this time.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "description": "The types of logs to retrieve.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "All",
+              "Error",
+              "Warning",
+              "Success"
+            ]
```

Comment:
still missing x-ms-enum

Reply:
commit https://github.com/Azure/azure-rest-api-specs/pull/16968/commits/b4801d025a17bf9c90cef971c41ce58ed441acc3 to address this issue